views/applications/_form.html

```diff
@@ -0,0 +1,13 @@
+<form action="{{action}}" method="post" enctype="multipart/form-data" class="form" novalidate>
+  <div class="form--field-container">
+    {{form.html | safe}}
+  </div>
+
+  <div class="form--actions">
+    <input type="submit" name="submit" value="{{button}}" class="form--submit">
+    {% if delete %}
+      <input type="submit" name="submit" value="{{config.actions.delete}}" class="form--submit">
+    {% endif %}
+    <a href="../" class="form--cancel">Cancel</a>
+  </div>
+</form>
```

Comment:
@Snugug but delete _is_ different for this form _and_ [users has a different way to currently handle deletes](https://github.com/punchcard-cms/punchcard/blob/master/lib/routes/users.js#L189)
The stories I'm writing for the webhooks will be copied when updating the Users section - but we should absolutely not attempt to make changes to the users section in this pr - that is very Out Of Scope.